<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddQuestion</class>
    <widget class="QWidget" name="AddQuestion">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>AddQuestion</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>40</y>
                    <width>191</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>14</pointsize>
                </font>
            </property>
            <property name="tabletTracking">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>Add new Question</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="questionLineEdit">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>120</y>
                    <width>201</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>160</y>
                    <width>61</width>
                    <height>17</height>
                </rect>
            </property>
            <property name="text">
                <string>Answer:</string>
            </property>
        </widget>
        <widget class="QPushButton" name="createButton">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>220</y>
                    <width>121</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="text">
                <string>Create</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="answerLineEdit">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>160</y>
                    <width>201</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>120</y>
                    <width>71</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>Question:</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
